nixos/modules/services/web-apps/loolwsd/default.nix

```diff
@@ -0,0 +1,245 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  user = "lool";
+  group = "lool";
+  dataDir = "/var/lib/lool";
+  cfg = config.services.loolwsd;
+  pkg = cfg.package;
+  corePkg = pkg.passthru.libreoffice;
+  subdir = corePkg.pname;
+  cmdLineOpts = {
```

Comment:
```suggestion
  cmdLineOpts = {
    server_name = cfg.proxy.hostname;
```
This should be empty if there is no hostname, so the option should have a default. Otherwise it can be set if only proxy.hostname is set.